lib/elastic_record/version.rb

```diff
@@ -0,0 +1,11 @@
+module ElasticRecord
+  class Version
+    include ElasticConnection
+
+    def self.es6?
+      @es6 ||= elastic_connection.json_get('/')
+        .dig('version', 'number')
+        &.start_with?('6.')
```

Comment:
Does ES 7.2 crash when _doc is provided? Curious what will happen during the rolling upgrade from 6.8 to 7.2

Reply:
It throws deprecation warnings and I think will eventually error in 8 (but I'm not certain).